compiler/noirc_driver/src/lib.rs

```diff
@@ -75,6 +75,10 @@ pub struct CompileOptions {
     #[arg(long, hide = true)]
     pub show_ssa_pass_name: Option<String>,
 
+    /// Only show the SSA and ACIR for the contract function with a given name.
+    #[arg(long, hide = true)]
+    pub show_contract_function_name: Option<String>,
```

Comment:
This is a bit long: perhaps `show_contract_fn`? Also, from the name of the argument, I would expect it to show the names of contract functions, not filter by name.

Reply:
I know what you mean, I had the same thought. I called it like this because of the `--show-ssa-pass-name` option.

Do you think we should remove `-name` from both?

Reply:
Shortened both, cc @asterite